README.md

```diff
@@ -273,13 +273,13 @@ Step 3: Convert relocatables to integers
 
 ```text
     1 (base[0] + 0) -> 1
-    2 (base[0] + 1) -> 4
-    3 (base[0] + 2) -> 7
-    4 (base[1] + 0) -> 8
+    2 (base[0] + 1) -> 2
+    3 (base[0] + 2) -> 3
+    4 (base[1] + 0) -> 4
```

Comment:
The calculation in brackets (like base[0] + 0) corresponds to the number to the left of it (in this case the relocated address). The number on the right of the arrow (->) is the element pointed by the address, not the result of the calculation. The notation can be a bit confusing, maybe this PR could propose a clearer way to display the process.

Reply:
that makes sence, will udpate it!

Reply:
I think we can improve the documentation making explicit that the arrow means the element pointed by the address in this case.